### PR TITLE
Docs for retriggering through issue comment

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: content/docs/Configuration/_index.md [hunk 1]
```diff
@@ -330,6 +330,13 @@ By default, no issue will be created so Packit doesn't unintentionally spam any
 By enabling this setting, Packit will either create a new issue or add a comment to an already existing issue.
 In that case, the issue will be reused and a new comment will be added.
 
+When a downstream job fails (Koji build or Bodhi update),
+the user can re-trigger the failed job through a comment like
```
Comment (TomasTomecek) [inline content/docs/Configuration/_index.md:334]:
```suggestion
you can re-trigger the failed job through a comment like 
```

**Review by TomasTomecek — APPROVED** [review]

File: content/docs/Configuration/_index.md [hunk 2]
```diff
@@ -336,6 +336,13 @@ By default, no issue will be created so Packit doesn't unintentionally spam any
 By enabling this setting, Packit will either create a new issue or add a comment to an already existing issue.
 In that case, the issue will be reused and a new comment will be added.
 
+When a downstream job fails (Koji build or Bodhi update),
+you can re-trigger the failed job through a comment like 
+`/packit koji-build` or `/packit create-update` in the opened issue.
+The Packit app has to be installed in the issue repository and the 
+user, which commented, needs to have write permissions
```
Comment (FrNecas) [inline content/docs/Configuration/_index.md:343]:
```suggestion
user, who commented, needs to have write permissions
```

File: content/docs/Configuration/_index.md [hunk 3]
```diff
@@ -336,6 +336,13 @@ By default, no issue will be created so Packit doesn't unintentionally spam any
 By enabling this setting, Packit will either create a new issue or add a comment to an already existing issue.
 In that case, the issue will be reused and a new comment will be added.
 
+When a downstream job fails (Koji build or Bodhi update),
```
Comment (FrNecas) [inline content/docs/Configuration/_index.md:339]:
```suggestion
When a downstream job (e.g. Koji build or Bodhi update) fails,
```

File: content/docs/guide.md [hunk 4]
```diff
@@ -298,6 +298,13 @@ or are run in a particular step as a hook (with no action run by default).
 Take a look at the tables [here](/docs/actions/) to know what actions are available for each job.
 
 
+### Users
+
+Packit can check for the user's rights to complete a job (this depends on the job type).
```
Comment (FrNecas) [inline content/docs/guide.md:303]:
```suggestion
Packit checks if the user has sufficient permissions to complete a job (this depends on the job type).
```

File: content/docs/guide.md [hunk 5]
```diff
@@ -298,6 +298,13 @@ or are run in a particular step as a hook (with no action run by default).
...
+For this reason the profile of a Packit user has to be accessible.
+In Github, as an example, this means that the membership of a user to its team has to be public (https://github.com/orgs/[org_name]/people/[user_name]).
```
Comment (FrNecas) [inline content/docs/guide.md:305]:
```suggestion
For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name]).
```

**Review by FrNecas — COMMENTED** [review]
A few wording suggestions

**Comment by majamassarini** [comment]
@FrNecas thanks!

File: content/docs/guide.md [hunk 6]
```diff
@@ -298,6 +298,13 @@ or are run in a particular step as a hook (with no action run by default).
 Take a look at the tables [here](/docs/actions/) to know what actions are available for each job.
 
 
+### Users
+
+Packit checks if the user has sufficient permissions to complete a job (this depends on the job type).
+For this reason the profile of a Packit user has to be accessible.
+For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name]).
```
Comment (FrNecas) [inline content/docs/guide.md:305]:
```suggestion
For example, on Github this means that the membership of a user to their team has to be public (see https://github.com/orgs/[org_name]/people/[user_name] ).
```
One last suggestion. Without the space, hugo renders the `)` as a part of the link.

**Review by FrNecas — APPROVED** [review]
Otherwise LGTM

**Pull request merged**